lib/xandra.ex

```diff
@@ -123,6 +137,10 @@ defmodule Xandra do
     * `:port` - (integer) the port of the Cassandra server to connect
       to. Defaults to `9042`.
 
+    * `:compressor` - (module) the compressor module to use for compressing and
+      uncompressing data. See the "Compression" section in the module
+      documentation. By default this option is not present.
```

Comment:
Do we really need to mention that it is not present? We are usually doing the opposite.